pkg/zfs/fs_util.go

```diff
@@ -23,3 +27,47 @@ func FullIdWithSnapshot(filesystemId, snapshotId string) string {
 	}
 	return filesystemId
 }
+
+func filterMountpoints(mountPrefix string, filesystem string, r *bufio.Reader) ([]string, error) {
+
+	mountPrefix = filepath.Join(mountPrefix, "dmfs", filesystem)
+
+	var mountpoints []string
+	for {
+		line, err := r.ReadString('\n')
+
+		if line != "" {
+			parts := strings.Split(line, " ")
+			if len(parts) >= 11 {
+				fsType := parts[8]
+				mountpoint := parts[4]
+				// mountedFS := parts[9]
+				// TODO: make this filter out only _our_ filesystem mounts (e.g. snapshot mounts)
```

Comment:
Long todo - are we doing this now or is it extra complexity that's not yet needed?

Reply:
thanks, removed, it's already filtering :)